homeassistant/components/fitbit/quality_scale.yaml

```
@@ -0,0 +1,81 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: The integration has no actions.
+  appropriate-polling:
+    status: done
+    comment: |
+      The integration uses polling to get data from Fitbit API every 30 minutes.
+  brands: done
+  common-modules:
+    status: done
+    comment: Coordinator is in coordinator.py and there is no base entity.
```

```suggestion
  common-modules: done
```
Would be fine as well imo